<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>From</class>
 <widget class="QWidget" name="From">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐播放器</string>
  </property>
  <widget class="QLabel" name="search_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索网易：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>140</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="search_pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="local_pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>本地</string>
   </property>
  </widget>
  <widget class="QLabel" name="timestart_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeend_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="none_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playmode_pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>mode</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>241</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
